tests/workflow_tests/test_deployment_workflows.py

```diff
@@ -82,3 +85,44 @@ def test_get_workflow_parameters(self):
             }
         }
         self.assertEqual(expected_params, execute_op_workflow.parameters)
+
+    def test_delete_botched_deployment(self):
+
+        from testenv import testenv_instance
+        storage_file_path = os.path.join(
+            testenv_instance.plugins_storage_dir,
+            'agent.json'
+        )
+
+        dsl_path = resource('dsl/botched_deployment.yaml')
+        _id = uuid.uuid1()
```

Comment:
we use uuid4 usually - to be honest, i have no idea why. i just read about it for the first time, doesn't really sound like it should matter. just mentioning in case you'd like to be consistent :p